### ISSUE TITLE
Capacity to add custom plugins without removing included plugins

### ISSUE THREAD

**Issue by appleshampooid** [llimllib/limbo#221]
It would be awesome if there were the ability (either via the existing `-pp` option, or a new one, to _append_ to the plugin path without remove all of the included plugins.

**Comment by llimllib** [comment]
agreed! I'm no longer adding features myself, but would absolutely accept a PR to do that

**Comment by llimllib** [comment]
#15 and #63 are related prior art here

**Comment by appleshampooid** [comment]
> agreed! I'm no longer adding features myself, but would absolutely accept a PR to do that

Awesome, I'm hoping to work on it.

**Comment by appleshampooid** [comment]
> > agreed! I'm no longer adding features myself, but would absolutely accept a PR to do that
> 
> Awesome, I'm hoping to work on it.

Hi @llimllib , any feedback on the PR?
https://github.com/llimllib/limbo/pull/222/
I've been running it for about a week on my instance of limbo.
